framework/berkeleydb/download.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import os
+import sys
+import subprocess
+import urllib.request as Download
+
+from framework.file.hash import FileHash
+
+TARBALL = "db-4.8.30.NC.tar.gz"
+DOWNLOAD_URL = "http://download.oracle.com/berkeley-db/" + TARBALL
+CHECKSUM = "12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef"
```

Comment:
This is in https://github.com/bitcoin/bitcoin/blob/c991b304dee368f506cfee27ddaa333f1f82c518/contrib/install_db4.sh#L21 already